src/example_scripts/get_submission_details.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+"""
+Overview
+========
+This script is designed to find submission information. It's main purpose is to demonstrate
+how to programmatically find submission information.
+
+Usage
+=====
+
+1. Run the script with a phase ID as the only argument to find all submissions on that phase.
+
+    ./get_submission_details.py 1
+
+2. Find the submission you would like detailed information about.
+
+3. Run the script again with the phase ID as the first argument and the submission ID as the second
+   argument to see a detailed submission object.
+
+    ./get_competition_details.py 1 1
+"""
+
+from sys import argv              # noqa: E402,E261  # Ignore E261 to line up these noqa
+from operator import itemgetter   # noqa: E402,E261
+from urllib.parse import urljoin  # noqa: E402,E261
+import requests                   # noqa: E402,E261
+from pprint import pprint         # noqa: E402,E261
+
+
+# ----------------------------------------------------------------------------
+# Configure these
+# ----------------------------------------------------------------------------
+CODALAB_URL = 'http://localhost/'
+USERNAME = 'admin'
+PASSWORD = 'admin'
+
+
+# ----------------------------------------------------------------------------
+# Script start..
+# ----------------------------------------------------------------------------
+PHASE_ID = None
+SUBMISSION_ID = None
+MODE = None
+
+if len(argv) > 1:
+    PHASE_ID = int(argv[1])
+    MODE = 1
+if len(argv) > 2:
+    SUBMISSION_ID = int(argv[2])
+    MODE = 2
+
+# Login
+login_url = urljoin(CODALAB_URL, '/api/api-token-auth/')
+resp = requests.post(login_url, {"username": USERNAME, "password": PASSWORD})
+if resp.status_code != 200:
+    print(f"Failed to login: {resp.content}")
+    print('Is the url correct? (http vs https)')
+    exit(-1)
+
+# Setup auth headers for the rest of communication
+token = resp.json()["token"]
+headers = {
+    "Authorization": f"Token {token}"
+}
+
+if MODE == 1:
```

Comment:
I think we could make this simpler and remove the "MODE" variable, do `if PHASE_ID and SUBMISSION_ID`?

Reply:
Ah yeah, can probably get rid of mode on the other script as well.